server/routes/notification.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { createNotification, getAllNotification, getUnreadNotification, updateNotification, updateAllUnreadNotification } = require("../controllers/notification");
+
+router.route("/create").post(protect, createNotification);
+router.route("/all").get(protect, getAllNotification);
```

Comment:
This can simply be `router.route('/')` with a `GET` request.

Reply:
updated